doc/classes/Vector2i.xml

```diff
@@ -124,6 +124,13 @@
 		<constant name="ONE" value="Vector2i(1, 1)">
 			One vector, a vector with all components set to [code]1[/code].
 		</constant>
+		<constant name="MIN" value="Vector2i(-2147483648, -2147483648)">
+			Negative equivalent of [constant MAX].
```

Comment:
Not strictly true, `INT_MIN` is not the same as `-INT_MAX`

Reply:
That's why it says "equivalent". I wanted to add a note that they are not equal, but not sure how to word it.

Reply:
It could be said to be a "minimum equivalent" but I'd not say "negative", perhsps "Min vector, equivalent to [constant MAX] but with [code]INT_MIN[/code]."

Reply:
> not sure how to word it.

Copy/paste and replace max by min? :thinking: :smile:
I don't think a reference to `MAX` is needed at all, they're next to each other in the docs anyway.

Reply:
Copy/paste would be the least confusing